app/features/ledger/ConcordiumLedgerClientMain.ts

```diff
@@ -448,6 +449,20 @@ export default class ConcordiumLedgerClientMain {
         );
     }
 
+    signCreatePlt(
+        transaction: UpdateInstruction<CreatePLTPayload>,
+        serializedPayload: Buffer,
+        path: number[]
+    ): Promise<Buffer> {
+        return signUpdateTransaction(
+            this.transport,
+            0x48,
+            path,
+            transaction,
+            serializedPayload
+        );
+    }
```

Comment:
This will not work, the data needs to be sent in batches. Have a look at the relevant doc for the instruction: https://github.com/Concordium/concordium-ledger-app/blob/b371047440bfb8dca8b3dd5100e3f15e44c9d134/governance-app/doc/ins_create_plt.md

Reply:
You can also have a look at the e2e test, might be easier to see how the data is transferred there:

https://github.com/Concordium/concordium-ledger-app/blob/b371047440bfb8dca8b3dd5100e3f15e44c9d134/tests/src/governance/createPlt.test.ts